CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 <!--
 Contains editorialized release notes. Raw release notes should go into `RELEASE-NOTES.txt`.
 -->
+## 20.1
+Explore smoother shopping experiences with our latest update! We've fixed an array of bugs, improved order creation flows, and enhanced campaign functionalities for non-Western Arabic numbering systems. Plus, enjoy streamlined order editing, updated campaign budgets, and a fresh barcode scanner icon. This version also resolves navigation crashes in the Hub Menu for iOS 18 beta users.
```

Comment:
This feels very AI-generated. I think I preferred the days of "bug fixes and updates."!

```suggestion
In this update we've improved the order creation flows, fixed several bugs in the edit order flows, and fixed a problem when starting a Tap to Pay on iPhone payment.
We've also made it easier to understand the budgets for your Braze ad campaigns, and added support for Braze ad creation in locales which don't use Western Arabic numerals.
```

We shouldn't mention iOS 18 because Apple get funny about it.
I added the Tap to Pay fix which isn't in the other release notes, but is in the build.

Reply:
Definitely better and cleaner 👍